doc/manual.rst

```diff
@@ -57,12 +58,30 @@ Echelle
 For echelle, you only have to specify the object location in a single
 order and the code will use its fractional position on all other orders.
 
-Here is an example from the DevSuite (VLT_manual)::
+Here are a few lines from the VLT/X-Shooter 
+example in the PypeIt DevSuite::
+
+    |                           filename    |       frametype |          ra |         dec |        target | dispname |   decker | binning |             mjd | airmass | exptime | arm | manual |
+    | XSHOO.2019-08-21T07:55:35.020.fits.gz |         science | 21:57:38.97 | -80:21:31.3 |     FRB190711 |  default |   1.2x11 |     1,1 | 58716.330266429 |    1.94 |   350.0 | VIS | 1:1181.8:3820.6:3. |
+    | XSHOO.2019-08-21T08:04:15.565.fits.gz |         science | 21:57:38.97 | -80:21:31.3 |     FRB190711 |  default |   1.2x11 |     1,1 | 58716.336291257 |   1.956 |   350.0 | VIS | 1:1181.8:3820.6:3. |
+
+The above will lay down a new trace at spatial=1181.8, 
+spectral=3820.6 pixel on detector 1 and use a FWHM 
+of 3 pixels.  It will also force an extraction at
+the same relative position for each echelle order.
+
+Coadd2D
+-------
+
+When using the :ref:`pypeit-coadd-2dspec` script, you
```

Comment:
Maybe you can clarify that for 2d codas there is no pypeit file and so this needs to be done in the .coadd2d file and point to the  vlt_xshooter_vis_manual.coadd2d in the dev suite. It is otherwise unclear where this goes.